### PR TITLE
usage.gif: add usage example

### PR DESCRIPTION
**Pull request by ghost**
Add usage example that will be shown in README

Closes https://github.com/coala/coala-atom/issues/20

### CODE REVIEW

**Comment by ghost** [comment]
omg, forgot to write issue id in commit message

**Comment by Nosferatul** [comment]
do it

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This package offers a plugin for the atom editor to use the code analysis
 software [coala](https://github.com/coala/coala) inside the
 editor.
 
+![Coala Atom](https://raw.githubusercontent.com/coala/coala-atom/master/i/usage.gif)
```
Comment (RohanVB) [inline README.md:7]: Coala -> coala

**Review by RohanVB — CHANGES_REQUESTED** [review]

**Comment by RohanVB** [comment]
gif looks alright 👍

**Comment by sils** [comment]
Uuuuuh this is awesome! Just change the c to a lower case one and we're good to go :)

**Comment by sils** [comment]
Is the gif already using https://github.com/coala/coala-atom/pull/63 ?

**Comment by RohanVB** [comment]
> Is the gif already using #63 ?

@sils that's by using the latest version of the base `linter` we built `coala-atom` on. It flags stuff differently.

**Pull request closed without merging**